lib/web_ui/dev/utils.dart

```diff
@@ -43,6 +43,7 @@ Future<int> runProcess(
     executable,
     arguments,
     workingDirectory: workingDirectory,
+    runInShell: io.Platform.isWindows,
```

Comment:
I'd copy the following line from the PR description and put it here:

> Running the process in a system shell. (otherwise it is not able to get dart from path. I used this tool to track the error https://docs.microsoft.com/en-us/sysinternals/downloads/procmon)

Reply:
done.